mocked-api/mocked/mocked_blood_dotation_centers.py

```diff
@@ -1,153 +1,7 @@
-mocked_data = []
+import json
 
-"""
-import strawberry
-from datetime import datetime, time
-from strawberry import relay
-from schema_types.blood_dotation_center_category import (
-    ENUM_BLOODDONATIONCENTER_CATEGORY,
-)  # noqa: E501
-from schema_types.weekday import ENUM_WEEKDAY
-from schema_types.tag import Tag
-from schema_types.address import Address
-from schema_types.contacts import Contacts
-from schema_types.work_schedule_item import WorkScheduleItem
-from schema_types.blood_donation_center import BloodDonationCenter
+with open("./mocked/bdc_mock_data.json", "r") as f:
```

Comment:
This code looks like it's executed without you having to call it.
I would wrap everythig that needs executed in methods and call them in main functions if needed.
When you leave code like this in the module, it will get executed at imports as well, maybe you don't want that.